styles/components/account/account-user-name.module.scss

```diff
@@ -0,0 +1,93 @@
+.account-user-name {
+	margin-bottom: 20px;
+	position: relative;
+
+	input[type="text"] {
+		overflow: auto;
+		padding-bottom: 3px;
+		font-family: Kanit;
+		width: 275px;
+		height: 55px;
+		position: relative;
+		background-color: transparent;
+		text-align: center;
+		font-size: 3.5rem;
+		font-weight: 500;
+		color: var(--secondary-color);
+	}
+
+	.user-name-input-save {
```

Comment:
described it in another comment but there's no need for this class name and the one below

Reply:
noted